Web/starter_project_barbies/pages/blog/my-blogs/[user-id].tsx

```diff
@@ -0,0 +1,70 @@
+import { NextPage } from "next";
+import { useRouter } from "next/router";
+import { useState, useEffect } from "react";
+import { Blog } from "@/types/blog";
+import BlogCard from "@/components/blog/BlogCard";
+
+
+// Fetch blogs from dummyData.json file 
+const fetchBlogs = async (): Promise<Blog[]> => {
+  const res = await fetch("/data/dummyData.json");
+  const data = await res.json();
+  
+  return data;
+};
+
+const MyBlogs: NextPage = () => {
+  const [blogs, setBlogs] = useState<Blog[]>([]);
+
+  // Get userID from router
+  const router = useRouter();
+  const userID = router.query["my-blogs"];
+  
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
for now, it is fine but realistically the data loading happens with state management (keep in mind that we will do that switch later on)